pmd-cataloger/build.gradle.kts

```diff
@@ -61,7 +61,11 @@ dependencies {
   implementation("com.google.code.gson:gson:2.3")
   implementation("com.google.guava:guava:28.0-jre")
   testImplementation("org.mockito:mockito-core:1.+")
-  testImplementation("junit", "junit", "4.12")
+  testCompileOnly("junit:junit:4.12")
+  testRuntimeOnly("org.junit.vintage:junit-vintage-engine:5.7.2")
+  testImplementation("org.junit.jupiter:junit-jupiter-api:5.7.2")
+  testImplementation("org.junit.jupiter:junit-jupiter-engine:5.7.2")
+  testImplementation("org.junit.jupiter:junit-jupiter-params:5.7.2")
```

Comment:
Switching the PMD Cataloger module from JUnit 4 to JUnit 5.
Adding the vintage engine allows us to put off rewriting all the existing tests.